ouroboros-consensus/src/Ouroboros/Consensus/Ledger/SupportsHD.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE ConstraintKinds #-}
+-- |
+
+module Ouroboros.Consensus.Ledger.SupportsHD (LedgerSupportsHD) where
+
+import           Ouroboros.Consensus.Ledger.Tables
+
+type LedgerSupportsHD l =
```

Comment:
I'm not sure about the term `HD`. What about `LedgerSupportTables`? Or `LedgerSupportsDiskStorage`? (even though we have an in-memory backend :/)

Reply:
Using `LedgerSupportsTables`